src/main/webapp/Forum-id.jsp

```diff
@@ -37,6 +37,10 @@
 <body>
 <h1><%=forum.getTitle()%></h1>
 <hr>
+  <form action="/createTrail.html" method="GET" id="AddArticle">
```

Comment:
Why is adding an article "GET" method?

Reply:
It is for directing to article creating page not creating an article. By passing origin of the click, the created article will be added to only the origin forum.